docs/source/optimization_ov.mdx

```diff
@@ -74,21 +74,22 @@ model = OVModelForCausalLM.from_pretrained(model_id, load_in_8bit=True)
 
 > **NOTE:** `load_in_8bit` is enabled by default for models larger than 1 billion parameters.
 
-For the 4-bit weight quantization we recommend using the NNCF API like below:
+For the 4-bit weight quantization you can use `load_in_4bit` option. The `quantization_config` can be used to controll the optimization parameters, for example:
+
 ```python
-from optimum.intel import OVModelForCausalLM
+from optimum.intel import OVModelForCausalLM, OVWeightQuantizationConfig
 import nncf
 
-model = OVModelForCausalLM.from_pretrained(model_id, load_in_8bit=False)
-model.model = nncf.compress_weights(
-        model.model,
-        mode=nncf.CompressWeightsMode.INT4_SYM,
-        ratio=0.8,
-        group_size=128,
-    )
-model.save_pretrained("compressed_model")
+model = OVModelForCausalLM.from_pretrained(
+    model_id,
+    export=True,
+    load_in_4bit=True,
```

Comment:
The [`load_in_4bit` and `load_in_8bit` arguments will soon be deprecated](https://github.com/huggingface/transformers/blob/69ca640dd6d52860d9e1ba5701ee06b0aedb0a1f/src/transformers/modeling_utils.py#L2965) in tranformers (since https://github.com/huggingface/transformers/pull/26610), I'll move it to the `quantization_config` in a coming PR. I can also update this one once merged. What do you think @AlexKoff88 ?

Reply:
From what I understood, the `quantization_config` itself will trigger the optimization workflow underneath. It sounds good to me. Let's merge and update since this PR just reflects what we have in the API.